tests/e2e/lib.sh

```diff
@@ -261,10 +261,42 @@ check_stackrox_logs() {
         die "StackRox logs were not collected. (Use ./scripts/ci/collect-service-logs.sh stackrox)"
     fi
 
+    check_for_stackrox_OOMs "$dir"
     check_for_stackrox_restarts "$dir"
     check_for_errors_in_stackrox_logs "$dir"
 }
 
+check_for_stackrox_OOMs() {
+    if [[ "$#" -ne 1 ]]; then
+        die "missing args. usage: check_for_stackrox_OOMs <dir>"
+    fi
+
+    local dir="$1"
+
+    if [[ ! -d "$dir/stackrox/pods" ]]; then
+        die "StackRox logs were not collected. (Use ./scripts/ci/collect-service-logs.sh stackrox)"
+    fi
+
+    local objects
+    objects=$(ls "$dir"/stackrox/pods/*_object.json || true)
+    if [[ -n "$objects" ]]; then
+        for describe in $objects; do
```

Comment:
nit:
```suggestion
        for object in $objects; do
```
or for pod in $objects